--- a/scons/flatpak_options.py	2024-09-06 15:44:34.326102896 -0500
+++ n/scons/flatpak_options.py	2024-09-06 15:50:41.539115973 -0500
@@ -32,13 +32,17 @@
 
 cxx_extra='-w -O3'
 
-pythoncmd='/app/bin/python3'
-pythonlibpath = '/app/lib'
-pythonincpath = '/app/include/python3.7m'
+pythoncmd='python3'
+pythonlibpath = '/app/lib/python3.13'
+pythonincpath = '/app/include/python3.13'
+pythonlibname='python3.13'
 
 boost_prefix=['/app/include','/app/lib']
+
 netcdf = 4
 netcdf_prefix=['/app/include','/app/lib']
+netcdf_libs = ['netcdf', 'netcdf_c++4']
+
 mpi_prefix=['/app/include','/app/lib']
 mpi_libs=['mpi_cxx']
 umfpack_prefix = ['/app/include','/app/lib']
