--- a/SConstruct	2026-09-22 17:09:11.925247420 -0500
+++ b/SConstruct	2026-09-24 17:28:59.864161923 -0500
@@ -1001,7 +1001,7 @@
         if sys.platform == 'darwin':
             env['launcher'] = "mpirun ${AGENTOVERRIDE} --gmca mpi_warn_on_fork 0 ${EE} "+hostoptionstr+" -np %N %b"
         else:
-            env['launcher'] = "mpirun ${AGENTOVERRIDE} --gmca mpi_warn_on_fork 0 ${EE} "+hostoptionstr+" --map-by node:pe=%t -bind-to core -np %N %b"
+            env['launcher'] = "mpirun ${AGENTOVERRIDE} --gmca mpi_warn_on_fork 0 ${EE} "+hostoptionstr+" --map-by node:pe=%t --bind-to core -np %N %b"
         if 'orte_version' in env:
             major,minor,point = [int(i) for i in env['orte_version'].split('.')]
             if major == 1 and minor < 10:
